### PR TITLE
feat(icons): added new icons - settlements, book, user, tag and shuffle

### PR DESCRIPTION
**Pull request by abhieshekumar**
### Added icons

- Book Icon
image

- Shuffle Icon
image

- Settlements Icon
image

- Tag Icon
image

- User Icon
image

### CODE REVIEW

**Comment by abhieshekumar** [comment]
Need to add two more icons -
- Transactions, 
- Edit
but there is a repetition of those.

https://razorpay.slack.com/archives/CMQ3RBHEU/p1672817419176859

File: .changeset/grumpy-mirrors-itch.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@razorpay/blade': major
```
Comment (saurabhdaware) [inline .changeset/grumpy-mirrors-itch.md:2]:
```suggestion
'@razorpay/blade': minor
```

Is there any breaking change in the PR? we can do minor release if not

**Review by saurabhdaware — COMMENTED** [review]
Looks good to me. Tiny comment regarding the changeset.

File: .changeset/grumpy-mirrors-itch.md (hunk 1 above)
Comment (abhieshekumar) [reply .changeset/grumpy-mirrors-itch.md]: No breaking changes @saurabhdaware . Changed it to minor, please check.

Comment (saurabhdaware) [reply .changeset/grumpy-mirrors-itch.md]: awesome. PR looks good to me. We'll discuss with the team how we want to go with icons with similar names and update here and in the thread.

**Pull request merged**